bitnami/contour/README.md

```diff
@@ -208,6 +214,18 @@ configInline:
   #   - "upstream_service_time"
   #   - "user_agent"
   #   - "x_forwarded_for"
+  #
+  # default-http-versions:
+  # - "HTTP/2"
+  # - "HTTP/1.1"
+  #
+  # The following shows the default proxy timeout settings.
+  # timeouts:
+  #   request-timeout: infinity
+  #   connection-idle-timeout: 60s
+  #   stream-idle-timeout: 5m
+  #   max-connection-duration: infinity
+  #   connection-shutdown-grace-period: 5s
```

Comment:
why do you need those values?

Reply:
The chart was merged into the bitnami repo with appVersion `1.5.1`. These settings in the `README.md`, `values.yaml`  and `values-production.yaml` were copied from the upstream kubernetes manifests you find here:
https://github.com/projectcontour/contour/blob/master/examples/contour/01-contour-config.yaml

Is it a bad idea to sync back the development process from the guys at projectcontour.io?

Reply:
Ok